environment-dev.yml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   - pandas==1.3.5
   - numpy>=1.16.5
-  - pyarrow>=1.0.0
+  - pyarrow>=6.*
```

Comment:
Don't use relative comparison and asterisk at the same time, conda gives a warning and discards the asterisk anyway.

Use `pyarrow>=6.0` to achieve the same effect.

Reply:
fixed